rules/tick.go

```diff
@@ -251,6 +251,7 @@ func getUniqOccupiedPoints(food []*pb.Point, snakes []*pb.Snake) []*pb.Point {
 
 func updateSnakes(game *pb.Game, frame *pb.GameFrame, moves []*SnakeUpdate) {
 	for _, update := range moves {
+		update.Snake.Latency = string(int64(update.Latency / time.Millisecond))
```

Comment:
I don't think this will work as you expect it to, this is just going to cast the number of milliseconds into what matches the bytes, see https://play.golang.org/p/syAhasID21y for an example. You probably want the following

```suggestion
		update.Snake.Latency = fmt.Sprint(update.Latency.Milliseconds())
```